docs/index.md

```diff
@@ -1,3 +1,11 @@
-# Shenanigan Wiki 🤸‍♀️️
+# Introduction
 
-📣️ Join Shenanigan [discord](http://www.she.energy/join) 📣️
+Shenanigan provides a competitive and motivating platform for athletes of all types utilizing fiscal incentives to drive personal improvement.
+
+We are doing this because we believe that athletes should be payed for the hard work that the put in to train and hone their craft. Shenanigan is built by athletes for athtletes to bring a different perspective to your everyday life.
+
+Athletes spend many hours of their time, sacrificing relationships, time with friends and family to be able to take their game to the next level. Few make it into the 1 percent of  professional play but most reach a wall where they must choose between their personal responsibilities and their sport.We offer an opportunity to intergrate the athletic grind into handling everday responsiblities. The platform we developed show cases athletes' skills and allows users to invest in their progression.Shenanigan uses NFTs to tokenize the athlete's progressive image and relies on a community of your pears and adversaries to generate liquidity  for the althletes' token. The Users(Particles) will charge  the Streamer(Ion)  based on if they are for or against them.
```

Comment:
spend countless hours of their lives to be able to take their game to the next level, often  sacrificing relationships and time with friends and family